drivers/watchdog/wdt_nxp_ewm.c

```diff
@@ -0,0 +1,199 @@
+/*
+ *
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_ewm
+
+#include <zephyr/drivers/watchdog.h>
+#include <zephyr/irq.h>
+#include <fsl_clock.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(wdt_nxp_ewm);
```

Comment:
log level can be an argument to the register macro, no need for LOG_LEVEL macro

Reply:
updated

Reply:
@EmilioCBen you removed the LOG_MODULE_REGISTER instead of the LOG_LEVEL, now there will be no logging

Reply:
Thanks, I have adjusted the change